src/main/resources/META-INF/plugin.xml

```diff
@@ -150,7 +150,12 @@
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
 
-        <action id="GitMachete.BaseDiscoverAction" class="com.virtuslab.gitmachete.frontend.actions.base.BaseDiscoverAction"/>
+        <!-- GitMachete.DiscoverAction is not added to any group because it's only available
+            from `Search Everywhere` and by link displayed in graph table when `machete` file
+            is empty or absent -->
+        <action id="GitMachete.DiscoverAction"
```

Comment:
Would be nice to add horizontal scroll if needed:
![image](https://user-images.githubusercontent.com/3383210/89320246-8775f000-d681-11ea-8a73-924f1faee642.png)

Reply:
It seems to be not so simple. [Issue #491](https://github.com/VirtusLab/git-machete-intellij-plugin/issues/491) created.